### PR TITLE
Remove temporary stronghold teleport

### PR DESCRIPTION
**Pull request by clouths**
Closes: [OpenPerpetuum/OP-Project#86](https://github.com/OpenPerpetuum/OP-Project/issues/86)

@Bler I would like to know if you approve the naming of the file I plan to use for this PR and future ones?
I'm using ISO 8601 (interchange format) at the beginning.

### CODE REVIEW

File: Staging_Dev/20190215T061300Z_remove_temp_stronghold_teleport.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+USE [perpetuumsa]
+GO
+
+delete from dbo.teleportdescriptions where sourcezone = 16 and targetzone = 0;
```
Comment (MikeJeffers) [inline Staging_Dev/20190215T061300Z_remove_temp_stronghold_teleport.sql]: The query here should be more strict as the intention is to delete one particular teleport - not all.
I know technically we want to remove ALL from this zone - but Deletes especially are dangerous to do with non-strict queries.

When in doubt - be specific.
In this case use in addition to the source/target other identifying properties of the teleport.

Also - Referencing other columns by ID is also problematic.
The source and targetzone IDs should be pulled from a query like:
`sourcezone = (SELECT id FROM dbo.zones WHERE name='-zonename-' AND note='-localizedzonename-')...`

**Review by MikeJeffers — CHANGES_REQUESTED** [review]

File: Staging_Dev/20190215T061300Z_remove_temp_stronghold_teleport.sql (hunk 1 above)
Comment (clouths) [reply Staging_Dev/20190215T061300Z_remove_temp_stronghold_teleport.sql]: Ok, I will be more specific for the next times. I'm updating this query at the moment to fix the issue.

File: Staging_Dev/20190215T061300Z_remove_temp_stronghold_teleport.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+USE [perpetuumsa]
+GO
+
+delete from dbo.teleportdescriptions
+where id = 403
```
Comment (MikeJeffers) [inline Staging_Dev/20190215T061300Z_remove_temp_stronghold_teleport.sql]: Please replace the ID lookup with name columns to find the correct row independent of auto incremented ids.

**Review by MikeJeffers — CHANGES_REQUESTED** [review]

File: Staging_Dev/20190215T061300Z_remove_temp_stronghold_teleport.sql (hunk 2 above)
Comment (clouths) [reply Staging_Dev/20190215T061300Z_remove_temp_stronghold_teleport.sql]: The table does not contain a column for the name, but I used description column instead.

**Review by MikeJeffers — APPROVED** [review]

**Comment by MikeJeffers** [comment]
@Bler for final review and merge

**Pull request merged**